modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -382,6 +380,7 @@ def _apply_func(partition, func, *args, **kwargs):  # pragma: no cover
     str
         The node IP address of the worker process.
     """
+    func, args, kwargs = deserialize(invocable)
```

Comment:
I ran `df.describe()` on a 20_000 x 10_000 dataframe of random ints (about as large as I could get before spilling to disk). On average, this PR finished the code in 6.029 seconds, while current master (af8600) finished in 6.058.

I do agree that there may be potential for lost parallelism here, and in this PR's [change to `deserialize` itself](https://github.com/modin-project/modin/pull/4715/files/456b885bc7d5756d8b89d12108cfb92b6c9c8663#diff-7ddcb1cabfe012c33c62919f2da1e7e4cb3d783907e45e5a1492c73d7bb377cdR221). However, since `Invocable` is now a simple `list` again, I should be able to revert this with little trouble.